src/audio/audio_reader.h

```diff
@@ -0,0 +1,68 @@
+//
+// Created by Yin, Weisu on 1/6/21.
+//
+
+#ifndef DECORD_AUDIO_READER_H_
+#define DECORD_AUDIO_READER_H_
+
+#include <vector>
+
+#include "../../include/decord/audio_interface.h"
+#ifdef __cplusplus
+extern "C" {
```

Comment:
do we need this in header file? better moved to .cc file if not required here

Reply:
Audio reader header is using type defined in swersample, so moving to cc won't compile. I moved this include to ffmpeg_common to keep ffmpeg lib organized.